src/components/admin/adminInquiry/AdminInquiry.tsx

```
@@ -0,0 +1,25 @@
+import { ADMIN_ROUTE } from '../../../constants/routes';
+import type { AdminInquiry } from '../../../models/inquiry';
+import ContentBorder from '../../common/contentBorder/ContentBorder';
+import * as S from './AdminInquiry.styled';
+
+interface AdminInquiryProps {
+  list: AdminInquiry;
+}
+
+export default function AdminInquiry({ list }: AdminInquiryProps) {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'AdminInquiry'. Consider to delete it or rename it.

'AdminInquiry' is defined here:


(lint/suspicious/noRedeclare)